config/environment.js

```diff
@@ -29,6 +29,9 @@ module.exports = function(environment) {
     'ember-simple-auth': {
       // Override the default `index` value
       routeIfAlreadyAuthenticated: 'home'
+    },
+    'ember-cli-mirage': {
+      enabled: true
```

Comment:
Any reason this is needed?
The setup that we have is that if we run `yarn start` without params it works with Mirage.
If we run `yarn start:dev` it proxies to conduit api to run against real data.

This is causing Mirage to always work even when proxying.